go-controller/pkg/allocator/persistentips/allocator.go

```diff
@@ -0,0 +1,44 @@
+package persistentips
+
+import (
+	ipamclaimsapi "github.com/k8snetworkplumbingwg/ipamclaims/pkg/crd/ipamclaims/v1alpha1"
+	ipamclaimslister "github.com/k8snetworkplumbingwg/ipamclaims/pkg/crd/ipamclaims/v1alpha1/apis/listers/ipamclaims/v1alpha1"
+	nadapi "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/allocator/ip/subnet"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kube"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+// Allocator acts on IPAMClaim events handed off by the cluster network
+// controller and allocates or releases IPs for IPAMClaims.
+type Allocator struct {
+}
+
+// NewPersistentIPsAllocator builds a new PersistentIPsAllocator
+func NewPersistentIPsAllocator(kube kube.InterfaceOVN, ipAllocator subnet.NamedAllocator) *Allocator {
```

Comment:
Since we are at `persistentips` we can just call this `NewAllocator` so we will have `persistentips.NewAllocator`

Reply:
Done.